inc/classes/rest/class-actions.php

```diff
@@ -0,0 +1,1044 @@
+<?php
+/**
+ * This will have code related to action scheduler required API.
+ *
+ * @package OneAccess
+ */
+
+namespace OneAccess\REST;
+
+use OneAccess\Plugin_Configs\Constants;
+use OneAccess\Traits\Singleton;
+use OneAccess\Utils;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_Error;
+
+/**
+ * Class Actions
+ */
+class Actions {
+
+	/**
+	 * REST API namespace.
+	 *
+	 * @var string
+	 */
+	private const NAMESPACE = 'oneaccess/v1';
+
+	/**
+	 * Batch size for user processing
+	 *
+	 * @var int
+	 */
+	private const BATCH_SIZE = 10;
+
+	/**
+	 * Results limit per request
+	 *
+	 * @var int
+	 */
+	private const RESULTS_LIMIT = 20;
+
+	/**
+	 * Use Singleton Trait
+	 */
+	use Singleton;
+
+	/**
+	 * Constructor
+	 */
+	protected function __construct() {
+		$this->setup_hooks();
+	}
+
+	/**
+	 * Setup hooks
+	 *
+	 * @return void
+	 */
+	protected function setup_hooks(): void {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register REST API routes
+	 *
+	 * @return void
+	 */
+	public function register_routes(): void {
+		/**
+		 * Route to add users to oneaccess_deduplicated_users table.
+		 */
+		register_rest_route(
+			self::NAMESPACE,
+			'/add-deduplicated-users',
+			array(
+				'methods'             => \WP_REST_Server::CREATABLE,
+				'callback'            => array( $this, 'add_deduplicated_users' ),
+				'permission_callback' => 'oneaccess_brand_site_to_governing_site_request_permission_check',
```

Comment:
As this method comes from a different file, and the namespace might be different there, so it's always a good practice to include the namespace from that file, or if there is no namespace required for this method, it is best practice to include `\` at the start of its use, indicating that it's not from this file.

Just want do double confirm, if that doesn't break anything. It might be required at other places too.